src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseRepository.java

```diff
@@ -174,6 +174,11 @@ public Claim saveClaimDocuments(
         return coreCaseDataService.saveClaimDocuments(authorisation, claimId, claimDocumentCollection);
     }
 
+    @Override
+    public Claim linkLetterHolder(Long claimId, String letterHolderId, String authorisation) {
```

Comment:
authorisation is not used at all, why are we passing?

Reply:
good pick